en/simulation/jmavsim.md

```diff
@@ -1,4 +1,17 @@
-# Software in the Loop (SITL) Simulation
+# jMAVSim with SITL
```

Comment:
FYI, the changes in this file are not supposed to be exhaustive. They are just enough to change the focus from "SITL" to "jMAVSim"